core/frontend/src/types/autopilot/px4/metadata-fetcher.ts

```diff
@@ -41,7 +41,11 @@ async function fetchPX4Metadata(): Promise<PX4ParametersMetadata[]> {
   try {
     metadata = await fetchPX4MetadataFromBoard()
   } catch (e) {
-    metadata = (await import('@/PX4-parameters/master/parameters.json')).parameters
+    const response = await fetch('/PX4-parameters/master/parameters.json');
```

Comment:
It doesn't look like this path exists on the built code. In order for this to work I think you'd need to move it to /public